stopcovid/sms/enqueue_outbound_sms.py

```diff
@@ -162,6 +162,7 @@ def publish_outbound_sms_messages(outbound_sms_messages: List[OutboundSMS]) -> A
                 "MessageBody": json.dumps(
                     {
                         "phone_number": phone,
+                        "messaging_service_sid": messages[0].messaging_service_sid,
```

Comment:
cool so the only change in this PR is adding `messaging_service_sid` so that the consumer of the outbound message queue can tell twilio which messaging service to use for each message?

Reply:
Correct!! That code is already live here: https://github.com/opus-training/dialog-engine/pull/142/files

I just missed the spot where we actually add the service to the outbound sms payload